__docs__/wonder-blocks-popover/popover.stories.tsx

```diff
@@ -833,3 +833,59 @@ export const WithCustomAriaDescribedBy = ({
         </View>
     );
 };
+
+export const InCorners = (args: PropsFor<typeof Popover>) => {
```

Comment:
**suggestion:** Could you please add a comment here to explain to devs how this example works? (including that they need to use `viewportPadding`)

Reply:
Yes good catch! Added 😄